backend/cloud/app.js

```diff
@@ -4,6 +4,8 @@ let express = require("express");
 let path = require("path");
 let { opengraphImage } = require("./services/imaging.js");
 let { Activity, Team, User } = require("./models");
+const config = require('../vue.config.js')

```

Comment:
this is not going to work, we are deploying only the `cloud`-folder as root. config must be  stored here and imported by `vue.config.js`  instead.